quickwit-index-config/src/config.rs

```diff
@@ -57,6 +57,14 @@ pub enum SortBy {
     DocId,
 }
 
+/// An index metadata carries all meta data about an index.
+//#[derive(Clone, Debug, Serialize, Deserialize)]
```

Comment:
I wish we removed that commented-out code before merging the PR t_t

Reply:
I think Paul removed it, but I accidentally restored it with a rebase + force push